src/Blocks/BlockSlab.h

```diff
@@ -88,7 +88,7 @@ class cBlockSlabHandler :
 	/** Returns true if the specified blocktype is one of the slabs handled by this handler */
 	static bool IsAnySlabType(BLOCKTYPE a_BlockType)
 	{
-		return ((a_BlockType == E_BLOCK_WOODEN_SLAB) || (a_BlockType == E_BLOCK_STONE_SLAB) || (a_BlockType == E_BLOCK_RED_SANDSTONE_SLAB));
+		return ((a_BlockType == E_BLOCK_WOODEN_SLAB) || (a_BlockType == E_BLOCK_STONE_SLAB) || (a_BlockType == E_BLOCK_RED_SANDSTONE_SLAB) || (a_BlockType == E_BLOCK_PURPUR_SLAB));
```

Comment:
This condition is getting too long, split it on separate lines, one per blocktype. Format properly:
```
return (
  (a_BlockType == E_BLOCK_WOODEN_SLAB) ||
  ( ... ) ||
  ...
);
```